src/Adapter/PlatformAdapter/Browser/Sdk/BrowserSdkAdapter.cpp

```diff
@@ -44,14 +48,12 @@ EMSCRIPTEN_BINDINGS(vgg_sdk)
     .function("setListenerKey", &VggSdk::setListenerKey)
     // misc
     .function("texts", &VggSdk::texts)
+    .function("makeImageSnapshot", &VggSdk::emMakeImageSnapshot)
     // doc
     .function("getElement", &VggSdk::getElement)
     .function("updateElement", &VggSdk::updateElement)
     .function("getDesignDocument", &VggSdk::designDocument)
     .function("valueAt", &VggSdk::designDocumentValueAt)
-    .function("addAt", &VggSdk::designDocumentAddAt)
-    .function("deleteAt", &VggSdk::designDocumentDeleteAt)
-    .function("updateAt", &VggSdk::designDocumentReplaceAt)
```

Comment:
Should we completely remove `addAt`/`deleteAt`/`updateAt` including its c++ functions? If not, what's the point for just deleting wasm interfaces?

Reply:
Yes, deleted.